docs/source/serialization.rst

```diff
@@ -193,3 +193,56 @@ Specify relations by intid:
     "relatedItems": [347127075, 347127076]
   }
 
+Walking the content
+-------------------
+
+The serialized content have some hints on how to move to the items around.
+
+Parent
+^^^^^^
+
+The parent is the folder where the page is inserted.
```

Comment:
The "parent" attribute points to the parent container of the current content object.